src/stores/organization/actions.ts

```diff
@@ -155,6 +157,17 @@ export function organizationsLoadMAPI(): ThunkAction<
             orgs.push(...rule.resourceNames);
           }
         });
+
+        // We now know what orgs they can get. We still need to fetch them
+        // to check if the orgs have a 'ui.giantswarm.io/original-organization-name'
+        // annotation.
+        const orgGetRequests = orgs.map((orgName) =>
+          getOrganization(client, user, orgName)()
```

Comment:
Users without ability to list OrgCRs now need to do 1 request for every org, to check if the orgCR has the annotation.